test/Migrator/swap_fixit.swift

```diff
@@ -0,0 +1,30 @@
+// RUN: rm -rf %t && mkdir -p %t && %target-swift-frontend -emit-migrated-file-path %t/swap_fixit.swift.result -enforce-exclusivity=checked -swift-version 3 -emit-sil -primary-file %s -o /dev/null -verify
```

Comment:
I don't think you want the '-verify' here (which is why the bots are complaining).